src/components/Navbar.tsx

```diff
@@ -107,20 +108,52 @@ const loadPageURL = (menuItem: any) => {
  */
 export default function Navbar() {
   const [selectedKeys, setselectedKeys] = useState<Array<string>>([]);
+  const [isDrawerOpen, setIsDrawerOpen] = useState(false);
```

Comment:
Decent way to implement drawer honestly. If there was a built-in component that could be better utilized but the built in navbar is not that good anyway.